models/dates.js

```diff
@@ -0,0 +1,30 @@
+function getDateRange(userStartDate, userEndDate) {
+  // If userStartDate is provided, use it; otherwise, use today's date
+  const startDate = userStartDate || formatDate(new Date());
+
+  // If userEndDate is provided, use it; otherwise, use a default value
+  const endDate = userEndDate || formatDate(new Date(new Date().setDate(new Date().getDate() + 7))); // 7 days from today as default
+
+  return {
+    startDate,
+    endDate
+  };
+}
+
+function getYesterdayDate() {
```

Comment:
Get all matches results from 1 day prior to today
Save to database using Postman